app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+  },
+  "feeChartTitle": {
+    "message": "Previsiones en Vivo de las Comisiones por Transacciones"
+  },
+  "forgetDevice": {
+    "message": "Olvidar a este dispositivo"
+  },
+  "functionType": {
+    "message": "Tipo de Función"
+  },
+  "gasUsed": {
+    "message": "Gas Usado"
+  },
+  "gasPriceExtremelyLow": {
+    "message": "Precio de Gas Excesivamente Bajo"
+  },
+  "gasPriceNoDenom": {
+    "message": "Precio de Gas"
+  },
+  "generatingTransaction": {
+    "message": "Generando la Transacción"
+  },
+  "getHelp": {
+    "message": "Pedir Ayuda."
+  },
+  "hardware": {
+    "message": "hardware"
+  },
+  "hardwareWalletConnected": {
+    "message": "Se ha conectado el monedero físico"
+  },
+  "hardwareWallets": {
+    "message": "Conectarle un monedero físico"
+  },
+  "hardwareWalletsMsg": {
+    "message": "Seleccionar un monedero físico que quieres usar con MetaMask"
+  },
+  "havingTroubleConnecting": {
+    "message": "¿Tienes problemas para hacer la conexión?"
+  },
+  "hexData": {
+    "message": "Datos de formato Hex"
+  },
+  "history": {
+    "message": "Historial"
+  },
+  "importUsingSeed": {
+    "message": "Importar usando la frase semilla de la cuenta"
+  },
+  "initialTransactionConfirmed": {
+    "message": "La red confirmó tu transacción inicial. Hazle clic en OK para volver."
+  },
+  "insufficientBalance": {
+    "message": "Saldo insuficiente."
+  },
+  "invalidSeedPhrase": {
+    "message": "Frase semilla no válida."
+  },
+  "keepTrackTokens": {
+    "message": "Estar al tanto con los tokens que has comprado a través de tu cuenta de MetaMask."
+  },
+  "learnMore": {
+    "message": "Más información"
+  },
+  "ledgerAccountRestriction": {
+    "message": "Hay que hacer uso de tu última cuenta antes de agregarle una nueva."
+  },
+  "liveGasPricePredictions": {
+    "message": "Previsiones en Vivo del Precio de Gas"
+  },
+  "menu": {
+    "message": "Menú"
+  },
+  "metamaskSeedWords": {
+    "message": "Palabras semilla de MetaMask"
+  },
+  "metamaskVersion": {
+    "message": "Versión de MetaMask"
+  },
+  "missingYourTokens": {
+    "message": "¿No ves tus tokens?"
+  },
+  "minutesShorthand": {
+    "message": "Min"
+  },
+  "nevermind": {
+    "message": "Olvídatelo"
+  },
+  "newNetwork": {
+    "message": "Red Nueva"
+  },
+  "rpcURL": {
+    "message": "Nuevo URL de RPC"
+  },
+  "showAdvancedOptions": {
+    "message": "Mostrar Opciones Avanzadas"
+  },
+  "hideAdvancedOptions": {
+    "message": "Ocultar Opciones Avanzadas"
+  },
+  "optionalChainId": {
+    "message": "ChainID (opcional)"
+  },
+  "optionalSymbol": {
+    "message": "Símbolo (opcional)"
+  },
+  "optionalNickname": {
+    "message": "Apodo (opcional)"
+  },
+  "newTotal": {
+    "message": "Nuevo Total"
+  },
+  "newTransactionFee": {
+    "message": "Nueva Comisión por Transacción"
+  },
+  "noConversionRateAvailable": {
+    "message": "No Hay Tasa de Conversión"
+  },
+  "notFound": {
+    "message": "No se localizó"
```

Comment:
"Find" === "Localizar" ? I'd use "Encontrar"

Reply:
That's an interesting one. According to the RAE, they're almost synonyms -- except 'localizar' seems to have the meaning of 'we know it exists, we just have to locate it', whereas 'encontrar' is more 'Look what I just found!'. I think 'encontrar' is used, for example, when police are investigating a crime scene and they're finding things... but 'localizar' would be used with a GPS monitor, for example. That's from my little corner of experience. 

However, I would say definitively that 'encontrar' is a lower register -- it's the more everyday word. Let's make it more accessible and use the more everyday word.